newIDE/app/src/VariablesList/VariableRow.js

```diff
@@ -29,12 +30,14 @@ type Props = {|
   onRemove: () => void,
   onAddChild: () => void,
   onChangeValue: string => void,
+  onResetToDefaultValue: () => void,
   children?: React.Node,
   muiTheme: Object,
   showHandle: boolean,
   showSelectionCheckbox: boolean,
   isSelected: boolean,
   onSelect: boolean => void,
+  type: String,
```

Comment:
use `string` rather than `String` (String is the JS String class, which is unnecessary. `string` is the primitive type).
Better, use a union type:
`type = 'object' | 'inherited' | 'instance'`

Could we rename these by the way to make the terminology reusable? 
'object' => 'parent' (or 'parent-variable')
'inherited' is fine
'instance' => empty string or 'variable', as you prefer. Try empty string, might be more elegant.

Reply:
Also finally found what was disturbing me! Could we rename "type" (which is super generic, and potentially confusing with "number"/"text"/"structure" that are the type of variables) to "origin"? :) 
Because this represents really "where is this variable coming from".